testzones/README.md

```diff
@@ -0,0 +1,47 @@
+This is work-in-progress.
+
+When loading `coredns` configuration this directory should be the working
+directory, else it will not find all included data files.
+
+
+In this directory the following items will be found:
+
+* data/
+  *A directory with data files. See README.md in that directory
```

Comment:
I think a space is missing after the asterisk.

Reply:
Fixed.